drivers/lcd/lcd_dev.c

```diff
@@ -118,7 +118,8 @@ static int lcddev_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
         FAR struct lcddev_area_s *lcd_area =
             (FAR struct lcddev_area_s *)arg;
         size_t cols = lcd_area->col_end - lcd_area->col_start + 1;
-        size_t row_size = cols * (priv->planeinfo.bpp >> 3);
+        size_t row_size = cols * (priv->planeinfo.bpp > 1 ?
```

Comment:
based on the code it looks more like `priv->planeinfo.bpp > 8` condition is appropriate, but I didn't dive deep into details

Reply:
Actually not. LVGL has multiple settings, like 1, 8, 16, 32. 1 is reserved for monochromatic displays, but it uses 1 byte per pixel memory model in buffer (strange yes).

Reply:
I meant that `8 >> 3` is the same as `1`. So maybe we can go with `((priv->planeinfo.bpp + 7) >> 3)`

Reply:
> I meant that `8 >> 3` is the same as `1`. So maybe we can go with `((priv->planeinfo.bpp + 7) >> 3)`

It would work also, but I do not know. It is a little bit less readable., don't you think.

Reply:
I'm fine with a current approach.